README.md

```
@@ -1,233 +1,181 @@
+<img src="first.svg" alt="Fibonacci flow" width="500">
+</div>
+
+```bash
+flowc -r flowrcli flowr/examples/fibonacci    # compile and run
+```
+
+The [first flow](book/first_flow/first_flow.md) section of the book walks you through it.
+
+## The Flow Tool Suite
+
+### <img src="flowc/assets/icons/flowc/icon.png" width="48" align="left"> flowc — Flow Compiler
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)